src/pudl/analysis/plant_parts_eia.py

```diff
@@ -1560,14 +1717,19 @@ def match_to_single_plant_part(
     """
     # select only the plant-part records that we are trying to scale to
     ppl_part_df = ppl[ppl.plant_part == part_name]
+    # select plant parts dictionary
+    if one_to_many:
+        plant_parts_dict = PLANT_PARTS_TRUE_GRAN
+    else:
+        plant_parts_dict = PLANT_PARTS
```

Comment:
you could just:
```suggestion
    # select plant parts dictionary
    plant_parts_dict = PLANT_PARTS_TRUE_GRAN if one_to_many else PLANT_PARTS
```

this is nit and an _almost_ entirely stylistic suggestion!